devtools/pyspy_pudl_etl.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/bash
+mkdir -p devtools/profiles/
```

Comment:
So if this is run from within the `devtools` directory we'll end up with a `devtools/devtools/profiles/` directory right?

Reply:
Yes, I figured it would be run from `/pudl`.

Reply:
I tweaked this a little to output to the devtools/profiles directory relative to the git repo root, and to standardize the output filenames.